bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/model/InverterData.java

```diff
@@ -82,6 +84,37 @@ default short getBatteryTemperature() {
         return Short.MIN_VALUE;
     }
 
+    default short getInverterTemperature1() {
+        return Short.MIN_VALUE;
+    }
+
+    default short getInverterTemperature2() {
+        return Short.MIN_VALUE;
+    }
+
+    default short getInverterWorkModeCode() {
+        return Short.MIN_VALUE;
+    }
+
+    @Nullable
+    default String getInverterWorkMode() {
+        switch (getInverterWorkModeCode()) {
```

Comment:
As suggested, maybe return an ENUM here instead of a string constant. Makes things more predictable.